src/platforms/android/configuration/integrations/fragment.mdx

```diff
@@ -9,21 +9,21 @@ categories:
 
 <Note>
 
-This is a preview API, available in `5.1.0-beta.1` and above. Features available in a preview API are still in progress and may have bugs. We recognize the irony. The API may be renamed, changed, or removed in a future version.
+This is a preview API, available in `5.1.0-beta.2` and above. Features available in a preview API are still in progress and may have bugs. We recognize the irony. The API may be renamed, changed, or removed in a future version.
 
 </Note>
 
 The `sentry-android-fragment` library provides [Fragment](https://developer.android.com/jetpack/androidx/releases/fragment) support for Sentry using the [FragmentLifecycleIntegration](https://github.com/getsentry/sentry-java/blob/a5f30b43b1dad2634ce020809f3b52e0d564a22a/sentry-android-fragment/src/main/java/io/sentry/android/fragment/FragmentLifecycleIntegration.kt). The source can be found [on GitHub](https://github.com/getsentry/sentry-java/tree/main/sentry-android-fragment/src/main/java/io/sentry/android/fragment).
 
-On this page, we get you up and running with Sentry's Fragment Integration, so that it will automatically add a breadcrumb for each Fragment's lifecycle.
+On this page, we get you up and running with Sentry's Fragment Integration, so that it will automatically add a breadcrumb for each Fragment's lifecycle and start a span out of the active transaction bound to the scope for each launch of a Fragment.
```

Comment:
```suggestion
On this page, we get you up and running with Sentry's Fragment Integration, so that it will automatically add a breadcrumb for each fragment's lifecycle and start a span out of the active transaction bound to the scope for each launch of a fragment.
```